module/scheduler.h

```diff
@@ -77,8 +79,9 @@ class Scheduler : public Module, ChannelHolder {
   unordered_map<uint32_t, BatchLog> all_logs_;
   BatchInterleaver local_interleaver_;
   DeterministicLockManager lock_manager_;
+  RemasterManager remaster_manager_;
 
-  unordered_map<TxnId, TransactionHolder> all_txns_;
+  TransactionMap all_txns_;
```

Comment:
@ctring since the remaster manager needs to know the keys of each txn, it needs a way to lookup the holders of lock only txns. Once the txn_replica_ids are assigned correctly this should behave the same for SH and MH, and LO's can be deleted from it once they are sent to the lock manager

The alternative would be if the remaster_manager kept it's own map from the replica id to txns, but this seems like a waste